tests/common.go

```diff
@@ -4042,6 +4043,39 @@ func GetAzureCredsFromEnv() (tenantID, clientID, clientSecret, subscriptionID, a
 	return tenantID, clientID, clientSecret, subscriptionID, accountName, accountKey
 }
 
+type NfsCredentials struct {
+	NfsServerAddress string
+	NfsPath          string
+	NfsSubPath       string
+	NfsMountOptions  string
+}
+
+func GetNfsCredsFromEnv() *NfsCredentials {
+	creds := &NfsCredentials{}
+
+	creds.NfsServerAddress = os.Getenv("NFS_SERVER_ADDR")
+	if creds.NfsServerAddress == "" {
+		err := fmt.Errorf("NFS_SERVER_ADDR environment variable should not be empty")
```

Comment:
Why not return the errror instead of handling them here?

Reply:
same as mentioned below, GetAzureCredsFromEnv ,GetAWSDetailsFromEnv() is handling error within the function.

Reply:
Okay fair enough